scripts/load_test_customer_data.sh

```diff
@@ -30,6 +30,9 @@ KOKU_PATH=$1
 START_DATE=$2
 END_DATE=$3
 
+# this is the default that's in koku.masu.config
+PVC_DIR=/var/tmp/masu
```

Comment:
This fixes an issue where `oc rsync` wasn't putting OCP data into the directory that the worker was expecting. So, this fixes it so the script now copies the files into the correct location.